README.md

```diff
@@ -253,6 +253,27 @@ See the full list of tasks with `i18n-tasks --help`.
 ✔ Reference keys (keys with `:symbol` values) are fully supported. These keys are copied as-is in
 `add/translate-missing`, and can be looked up by reference or value in `find`.
 
+#### Unexpected normalization
```

Comment:
Great draft, I'll think about the layout and if I want to move it around a bit. Thank you 🙂